{{cookiecutter.project_slug}}/clients/mobile/react-native/src/services/user/api.ts

```diff
@@ -34,12 +34,22 @@ const logout = createCustomServiceCall(async ({ client }) => {
   return client.post(`/logout/`)
 })
 
+const patchUser = createCustomServiceCall({
+  inputShape: partialUserShape,
+  outputShape: userShape,
+  cb: async ({ client, slashEndingBaseUri, input, utils }) => {
+    const { id, ...body } = utils.toApi(input)
+    const res = await client.put(`${slashEndingBaseUri}${id}/`, body)
+    return utils.fromApi(res.data)
+  },
+})
+
```

Comment:
Why not the built-in `update` ?
It is a patch and I think it does exactly this what you're doing in the custom call

Reply:
also remove the partial user shape